dsl/src/main/scala/org/http4s/dsl/impl/Statuses.scala

```diff
@@ -63,6 +63,8 @@ trait Statuses {
   val UnsupportedMediaType: Status.UnsupportedMediaType.type = Status.UnsupportedMediaType
   val RangeNotSatisfiable: Status.RangeNotSatisfiable.type = Status.RangeNotSatisfiable
   val ExpectationFailed: Status.ExpectationFailed.type = Status.ExpectationFailed
+  val UnprocessableContent: Status.UnprocessableContent.type = Status.UnprocessableContent
+  @deprecated("now called UnprocessableContent", since = "0.23.31")
```

Comment:
Would we like to preserve this deprecation in 1.0?

Reply:
My default is yes, but I've been told I hang on to deprecations too long.